src/cryptoadvance/specter/templates/includes/addresses-table.html

```diff
@@ -0,0 +1,503 @@
+        if (!self.listType) {
+            self.emptyRow.children[1].innerText = "Failed to fetch addresses...";
+            return
+        }
+
+        const url = `{{ url_for('wallets_endpoint.addresses_list', wallet_alias='WALLET_ALIAS') }}`.replace("WALLET_ALIAS", self.wallet);
+
+        self.export.href = `{{ url_for('wallets_endpoint.addresses_list_csv', wallet_alias='WALLET_ALIAS') }}`.replace("WALLET_ALIAS", self.wallet);
+        self.export.href += `?onlyCurrentType=${self.onlyCurrentTypeSwitch.checked}&addressType=${self.listType}&sortby=${self.sortby}&sortdir=${self.sortdir}`;
+
+        if (self.listType == "receive") {
+            self.switchText.innerText = "Only receive addresses: "
+        } else if (self.listType == "change") {
+            self.switchText.innerText = "Only change addresses: "
+        } else {
+            self.switchText.innerText = "";
+        }
+
+        let callId = self.callId;
+        var formData = new FormData();
+        formData.append('idx', self.idx);
+        formData.append('limit', self.limit);
+        formData.append('addressType', self.listType);
+        formData.append('sortby', self.sortby);
+        formData.append('sortdir', self.sortdir);
+
+        try {
+            const response = await fetch(
+                url,
+                {
+                    method: 'POST',
+                    body: formData
+                }
+            );
+
+            if (callId != self.callId) {
+                return;
+            }
+            if (response.status != 200) {
+                showError(await response.text());
+                return;
+            }
+            const jsonResponse = await response.json();
+
+            let addressesList = JSON.parse(jsonResponse.addressesList);
+
+            for (let addr of addressesList) {
+                /*let rowClass = "addr-row";
+
+                if (addr.used) rowClass += ' unconfirmed';
+
+
+                let addrRow = `
+                    <tr class="${rowClass}">
+                        <td>#${addr.index}</td>
+                        <td>${addr.address}</td>
+                        <td><address-label data-address="${addr.address}" data-label="${addr.label}" data-wallet="{{ wallet_alias }}"></address-label></td>	
+                        <td>${addr.used ? 'Yes' : 'No'}</td>
+                        <td>${addr.utxo}</td>
+                        <td>${addr.amount}</td>
+                        <td></td>
+                    </tr>`;*/
+
+                let addrRow = `
+                        <address-row 
+                            data-btc-unit="${self.btcUnit ? self.btcUnit : 'btc'}"
+                            data-price="${self.price ? self.price : 0}"
+                            data-symbol="${self.symbol ? self.symbol : ''}"
+                            data-address='${JSON.stringify(addr).replace(/[\/\(\)\']/g, "&apos;")}'
```

Comment:
Needs here the same fix as in #861

Reply:
Fixed. Maybe it can be interesting an eventual common component for both transactions and addresses UI.